code/modules/hivemind/mobs.dm

```diff
@@ -0,0 +1,916 @@
+
+///////////Hive mobs//////////
+//Some of them can be too tough and dangerous, but they must be so. Also don't forget, they are really rare thing.
+//Just bring corpses from wires away, and little mobs is not a problem
+//Mechiver have 1% chance to spawn from machinery. With failure chance calculation, this is very raaaaaare
+//But if players get some of these 'big guys', only teamwork, fast legs and trickery will works fine
+//So combine all of that to defeat them
+
+
+/mob/living/simple_animal/hostile/hivemind
+	name = "creature"
+	icon = 'icons/mob/hivemind.dmi'
+	icon_state = "slicer"
+	health = 20
+	maxHealth = 20
+	melee_damage_lower = 5
+	melee_damage_upper = 10
+	faction = "hive"
+	attacktext = "bangs with his head"
+	universal_speak = TRUE
+	speak_chance = 5
+	var/malfunction_chance = 5
+	var/ability_cooldown = 30 SECONDS
+	var/list/target_speak = list()			//this is like speak list, but when we see our target
+
+	//internals
+	var/obj/structure/hivemind_machine/master
+	var/special_ability_cooldown = 0		//use ability_cooldown, don't touch this
+
+
+	New()
+		..()
+		//here we change name, so design them according to this
+		name = pick("strange ", "unusual ", "odd ", "undiscovered ", "an interesting ") + name
+
+//It's sets manually
+/mob/living/simple_animal/hostile/hivemind/proc/special_ability()
+	return
+
+
+/mob/living/simple_animal/hostile/hivemind/proc/is_on_cooldown()
+	if(world.time >= special_ability_cooldown)
+		return FALSE
+	return TRUE
+
+
+//simple shaking animation, this one move our target horizontally
+/mob/living/simple_animal/hostile/hivemind/proc/shake_the(var/atom/target)
+	var/init_px = target.pixel_x
+	animate(target, pixel_x=init_px + 10*pick(-1, 1), time=1)
+	animate(pixel_x=init_px, time=8, easing=BOUNCE_EASING)
+
+
+//That's just stuns us for a while and start second proc
+/mob/living/simple_animal/hostile/hivemind/proc/mulfunction()
+	stance = HOSTILE_STANCE_IDLE //it give us some kind of stun effect
+	target_mob = null
+	walk(src, 0)
```

Comment:
NONE

Reply:
We haven't it here. I replace it with FALSE